base/Dockerfile

```diff
@@ -100,6 +100,18 @@ RUN apk add --no-cache \
     && apk del .build-dependencies \
     && rm -rf /usr/src/arp-scan
 
+# PicoTTS - it has no specific version - unfortunately we'll have to use master directly
+RUN apk add automake autoconf git libtool popt-dev git build-base \ 

```

Comment:
These are build dependencies, they need to be installed virtual and cleaned up after the build has finished. Also, please note, we don't have any package cache, so you'll need to use `--no-cache` as well.

Reply:
Added argument using the full commit hash.